about.html

```diff
@@ -46,7 +46,7 @@ <h1 class="title bold">Leaders Week</h1>
                 <p class="description">
                     A joyful celebration in the value of openess and sharing,
                     creating a positive change with people from over 10 countries
-                    is taking place in July, in Benin.
+                    is taking place in July, in Cotonou, Benin.
                 </p>
                 <div class="hero-event">
```

Comment:
### I write to you the review for this part at these lines here as I can not see all the code of the page 😊

- [ ]  Kindly it will be better if you fix the responsiveness at the main page as there is a fault in the first section I think it is needed some height to make it like the Behnace in the responsive, so let's try to fix the responsive at this part to let the user be able to see the project from all sizes of screen 💯🚀

> Check below to see the difference 😊⤵️
> If you can not see the image clearly just click on it to open a new tab in the browser😊

| Original design | Your design| 
| --- | --- |
|   ![image](https://github.com/lily4178993/leaders-week/assets/83360636/01bf5db5-c831-47a6-980c-52adcbf08230)  |     ![image](https://github.com/lily4178993/leaders-week/assets/83360636/f96ad285-cb8f-4dfa-a841-47e818a0c7d2)   |